builder/Dockerfile

```diff
@@ -5,6 +5,9 @@ MAINTAINER CenturyLink Labs <clt-labs-futuretech@centurylink.com>
 RUN wget -nv https://get.docker.com/builds/Linux/x86_64/docker-1.3.3 -O /usr/bin/docker && \
   chmod +x /usr/bin/docker
 
+ENV LDFLAGS='-s' \
```

Comment:
The only minor downside I see with this approach is that the defaults are now up in the Dockerfile.